gurubu-client/src/app/components/page/features.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+
+const Features = () => {
+  return (
+    <section className="features-section">
+      <div className="features-heading">
+        <div className="features-heading__pillbox">Features</div>
+        <p className="features-heading__title">No registration, share the link with your teammates and get started!</p>
+        <p className="features-heading__subtext">It is enough to create a room and share the link with your friends. We do not store any of your information!</p>
+      </div>
+      <div className="features-spotlight">
+        <Image src={'/screen-mockup.svg'} width={768} height={512} alt="An example of in app visuals" />
```

Comment:
we can write the gurubu
```suggestion
        <Image src={'/screen-mockup.svg'} width={768} height={512} alt="gurubu" />
```